{{cookiecutter.project_slug}}/installer.py

```diff
@@ -0,0 +1,70 @@
+#!/bin/python
+
+import subprocess as sp
+from argparse import ArgumentParser, BooleanOptionalAction
+
+
+def condacmd(cmd : str):
+    '''Wrapper for conda shell commands.
+    
+    '''
+    proc =  sp.run(f"conda {cmd}", shell=True, stdout=sp.PIPE, stderr=sp.STDOUT)
+    outstream = ''
+    if not proc.stdout is None:
```

Comment:
From my guessing of the intended behaviour here, that is
* decode with utf-8 the stdout, stderr
* pass stderr to stdout (no capture), capture stdout and additionally forward it to stdout (print)

 perhaps, the following is shorter: 
```
output = sp.run(f"conda {cmd}", shell=True, stdout=sp.PIPE, stderr=sp.STDOUT, stdout=sp.PIPE, check=True,encoding='utf-8').stdout
print(output)
return output
```
assuming stdout will only be None if stdout is not captured, since stdout=sp.PIPE, the check `if not proc.stdout is None` is not really needed. stderr on the contrary will always be None, so the current call to encoding will never be called (presumably).

Reply:
Hm I see that my attempt to catch the error messages from conda has serious flaws. The problem with @cosunae 's suggestion is that in case of a conda error the check=True will raise an error and the only thing you get in the traceback is  `returned non-zero exit status 1` which is not very helpful. So new suggestion:

`proc = sp.run(f"conda {cmd}", shell=True, stdout=sp.PIPE, stderr=sp.STDOUT, stdout=sp.PIPE, check=True, encoding='utf-8')
try:
    proc.check_returncode()
    print(proc.stdout)
except subprocess.CalledProcessError as e:
    print(e.output)
    sys,exit(1)`

Reply:
for reference, I think if you capture only stdout but not stderr, when raising an exception in case of failure (using check=True), it will still print the error to stdout, I think 
```
proc = sp.run(f"conda {cmd}", shell=True, stdout=sp.PIPE, check=True, encoding='utf-8')
```